web-bluetooth/read-characteristic-value-changed.js

```diff
@@ -15,8 +15,8 @@ function onReadBatteryLevelButtonClick() {
 
 function requestDevice() {
   log('Requesting Bluetooth Device...');
-  return navigator.bluetooth.requestDevice(
-    {filters: anyNamedDevice(), optionalServices: ['battery_service']})
+  return navigator.bluetooth.requestDevice({
+      acceptAllDevices: true, optionalServices: ['battery_service']})
```

Comment:
Would https://github.com/GoogleChrome/samples/pull/458/commits/9533907bcc3c6d3e34016fc3dd8d45f553e9d160 be enough?